src/contexts/sendSearchRequest.ts

```diff
@@ -44,12 +44,18 @@ export const sendSearchRequest = async ({
     };
   });
 
+  // reranking only works in English
+  if ((language != "eng") && (language != "auto")) {
```

Comment:
Re-ranking does not work on non-english languages, so disabling in this case.

Reply:
Nit: General best practice is to use `!==` instead of `!=` unless you desire a truthy comparison.

Reply:
On second thought, now I'm curious about this. Why does reranking only work in English? Isn't `language` used to determine the summary language? I'm surprised it has any impact on reranking, which I believe is a search-specific feature.

Reply:
Excellent point. So I think the reranking doesn't work if the "piece of text" returned from the search is non-english, but "lang" in vectara-answer just controls the summarization language. So all of what I did here is actually not doing anything...
So maybe I just change the 50 to 100 (that should still be useful when reranking), but no need to change the rerank flag. Instead we can emphasize this in the documentation. wdyt?

Reply:
I think that your proposal makes sense. Do our docs make any note of the incompatibility between reranking and non-English results? If so, maybe we can link to them. If not, I think we should file an issue requesting we add these details to the docs.

Reply:
Okay so made some updates:
1. Removed the whole language thing
2. Added rerank_results_num to specify number of results for reranking (default 100)